jcli/src/jcli_lib/rest/v1/vote/mod.rs

```diff
@@ -0,0 +1,58 @@
+use crate::jcli_lib::rest::{Error, RestArgs};
+use crate::jcli_lib::utils::OutputFormat;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct AccountVotes {
+    #[structopt(flatten)]
+    args: RestArgs,
+
+    #[structopt(flatten)]
+    output_format: OutputFormat,
+
+    /// Account address to filter votes
+    #[structopt(short, long)]
+    account: String,
+
+    /// Id of the voteplan for which we want to list proposals
+    /// the account voted for
+    #[structopt(short, long)]
+    voteplan_id: String,
```

Comment:
The name for this option is `--vote-plan-id` [elsewhere](https://github.com/input-output-hk/jormungandr/blob/99fa3c7d8f1120556916cbe55dcc94574fac04ec/jcli/src/jcli_lib/certificate/new_vote_cast.rs#L15-L15), let's make it consistent.